src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/GATKSVVCFHeaderLines.java

```diff
@@ -114,11 +116,15 @@ private static void addSymbAltAlleleLine(final VCFSimpleHeaderLine line) {
                     1,
                     VCFHeaderLineType.String,
                     "Comma-delimited list of external copy number calls that overlap with this variant in format ID:CN:CNQ"));
+
+            addInfoLine(new VCFInfoHeaderLine(GATKSVVCFConstants.LINK,
+                    VCFHeaderLineCount.UNBOUNDED,
+                    VCFHeaderLineType.String,
+                    "ID(s) of other record(s) linked to current record"));
         }
         
         // for variants-detected from assembly
-        // todo: create an alternate assembly file and link to it with breakpoint IDs according to the VCF spec
-        {
+        {// todo: create an alternate assembly file and link to it with breakpoint IDs according to the VCF spec
```

Comment:
Do we already do this? Do you know what needs to happen to make this spec-compliant?

Reply:
OK! Just checking to see if you wanted to keep this todo comment around or if it could be resolved.